src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5117,6 +5117,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ExplicitReadOnlyAttr" xml:space="preserve">
     <value>Do not use 'System.Runtime.CompilerServices.IsReadOnlyAttribute'. This is reserved for compiler usage.</value>
   </data>
+  <data name="ERR_ExplicitIsByRefLikeAttr" xml:space="preserve">
+    <value>Do not use 'System.Runtime.CompilerServices.IsByRefLikeAttribute'. This is reserved for compiler usage.</value>
```

Comment:
>Do not use [](start = 11, length = 10)

Is this error about declaring type with this name? Perhaps, the error should say "Do not declare type named ..." then?